packages/nimble-components/src/table/models/tests/sort-operations.spec.ts

```diff
@@ -0,0 +1,99 @@
+import type { Row as TanStackRow, SortingFn } from '@tanstack/table-core';
+import { TableColumnSortOperation } from '../../../table-column/base/types';
+import { getTanStackSortingFunction } from '../sort-operations';
+import type { TableFieldValue } from '../../types';
+
+describe('Sort Operations', () => {
+    function createFakeRow(
+        value: TableFieldValue
+    ): TanStackRow<TableFieldValue> {
+        const row = jasmine.createSpyObj<TanStackRow<TableFieldValue>>(
+            'TanStackRow',
+            ['getValue']
+        );
+        row.getValue.and.returnValue(value);
+        return row;
+    }
+
+    const basicNumericSortOrder = [
```

Comment:
Each of these `*SortOrder` arrays is ordered from smallest value to largest value. The tests call the sort operation on each index of each array with every index in the same array and verify that it returns the correct value. I had originally gone down the path of writing a bunch of individual tests cases for things like _undefined is equal to undefined_, _undefined is less than null_, _null is greater than undefined_, etc., but the list of tests was getting really long, which made it difficult to make sure we covered all the cases we wanted.

Reply:
>Each of these *SortOrder arrays is ordered from smallest value to largest value. The tests call the sort operation on each index of each array with every index in the same array and verify that it returns the correct value.

I think that would be a useful comment somewhere in this file.